src/pages/experiments/[experimentId]/plots-and-tables/volcano/index.jsx

```diff
@@ -155,10 +179,10 @@ const VolcanoPlot = ({ experimentId }) => {
 
     const date = moment.utc().format('YYYY-MM-DD-HH-mm-ss');
     const fileName = `de_${experimentId}_${cellSet}_vs_${compareWith}_${date}.csv`;
-    const disabled = plotData.length === 0 || loading || _.isEmpty(spec.spec) || error;
```

Comment:
so if the spec is empty initially while loading everything, will the export button be disabled?

Reply:
The CSV link does not depend on the spec, so I remove it. That allows the users to download the data without haveing to load for the spec to load. `loading` and `error` is renamed to `diffExprLoading` and `diffExprError` respectively.